specs/manage_tokens_list/01_main.rst

```diff
@@ -0,0 +1,95 @@
+==========================================================
+Manage Tokens - Selection List
```

Comment:
It seems that "selection list" is not a scenario per se, rather what I would expect to see in the scenarios:

- Display Selected Tokens
  - part of the use case on the main screen; should include handling of "default" (preselected) tokens
- Select Token
  - extension: search token
  - extension: pagination
- Deselect Token
- Update Token Information (automatic update)
  - extension: what if a previously selected token gets removed from our backend token list?

Reply:
search tokens is a different issue (#56) 
default tokens is a different issue (#57)

Reply:
I added my comments. In short, this spec is currently Android-oriented. We should discuss if we should have same ui/ux for both platforms, or different ui/ux with different spect, or more high-level user story spec with low-level technical details specs for both platforms.